Sources/Preprocessors/DefaultAnimationPreprocessor.swift

```diff
@@ -249,9 +259,9 @@ class DefaultAnimationPreprocessor: BasePreprocessor {
       if animators.contains(where: { $0.canAnimate(view: toView, appearing: true) || $0.canAnimate(view: fromView, appearing: false) }) {
         defaultAnimation = .none
       } else if inNavigationController {
-        defaultAnimation = presenting ? .push(direction:.left) : .pull(direction:.right)
+        defaultAnimation = presenting ? .push(direction:.leading) : .pull(direction:.trailing)
       } else if inTabBarController {
-        defaultAnimation = presenting ? .slide(direction:.left) : .slide(direction:.right)
+        defaultAnimation = presenting ? .push(direction:.leading) : .pull(direction:.trailing)
```

Comment:
Why did we change from `.slide` to `.push` here?

Reply:
oops, you are right. My bad, will fix it.

Reply:
Fixed